src/Components/WebAssembly/Build/src/targets/ServiceWorkerAssetsManifest.targets

```diff
@@ -36,7 +52,52 @@
 
   </Target>
 
-  <Target Name="_ComputeServiceWorkerAssetsManifestFileHashes">
+  <Target Name="_ComputeManifestIntegrity"
+    Condition="'$(ServiceWorkerAssetsManifest)' != ''"
+    BeforeTargets="_BlazorStaticWebAssetsCopyGeneratedFilesToOutputDirectory;_GzipCompressBlazorApplicationFiles">
```

Comment:
Should `_BrotliCompressBlazorApplicationFiles` also be in the `BeforeTargets` list? Not sure why we'd treat gzip differently from brotli for this purpose.

Reply:
Gzip happens at build time, Brotli happens at publish time, putting _Brotli there causes issues with no build as it forces the recomputation of a bunch stuff and ends up producing a wrong manifest.